shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -335,6 +352,120 @@ - (BOOL)handleKeyEvent:(NSEvent*)event {
   return [_textInputContext handleEvent:event];
 }
 
+#pragma mark NSResponder
+
+- (void)keyDown:(NSEvent*)event {
+  if (!self.flutterViewController) {
+    return;
+  }
```

Comment:
Here and below:

I'm not quite sure what the style guide says for this, but this isn't strictly necessary. Sending a message to `nil` is just a no-op in Objective C.

Reply:
(It does not throw an NPE, like you might expect calling a method on a null object in other langauges would)